Adding Access view

AlejoM1908 opened the pull request:
Adding the access allowed or denied view for nfc reading

Commit:
Creating the view for the access with backend info

Changes:
- Adding all new required icons (profile_icon needs a change)
- Adding the widget tree for view (modularity needed)

Commit:
Adding the final fixes to access view ux

Changes:
- Adding the correction of the view from mockups
- Adding functionality to change between allowed and denied access

Commit:
Fixing error in access view

Changes:
- Deleted one definition of a variable that generated error in the view

Andryut commented on lib/main.dart:
```diff
@@ -1,5 +1,4 @@
-//import 'dart:html';
-
+// FLutter imports
```
Me parece que este comentario es sobre explicativo

AlejoM1908 replied:
claro, pero la idea de las tag en los import es permitir que alguien que vea el código pueda hacerse una idea rápida sobre los componentes de cada view y las librerias que se usan o los modelos de los desarrolladores

Andryut commented on lib/util/colors.dart:
```diff
@@ -4,6 +4,7 @@ import 'dart:ui';
 const lightBlue = const Color(0xFF218BFB);
 const textColor = const Color(0xFF303432);
-const orange = const Color(0xFFFF8F22);
+const white = const Color(0xFFFFFFFF);
```
Este va a modo de pregunta, Flutter no tiene constantes ya definidas para estos colores?

AlejoM1908 replied:
SI, pero definirlos de este modo es un poco más visual para otros compañeros tocando el código y permite más flexibilidad e la escala del ARGB

Andryut commented on lib/util/routes.dart:
```diff
@@ -1,7 +1,8 @@
 import 'package:flutter/material.dart';
 import 'package:go_mobile/views/nfc_view.dart';
+
+// Project imports
```
Tal vez acá si hace sentido y hace ver el código más ordenado

Andryut commented on lib/views/access_view.dart:
```diff
@@ -0,0 +1,80 @@
+import '../util/icons.dart' as appIcons;
+
+// ignore: must_be_immutable
+class AccessView extends StatefulWidget {
```
¿Dónde se llama este widget?

AlejoM1908 replied:
el uso de este widget esta en el nfcView en la respuesta del query al backend

Andryut replied:
son estrictamente necesarios los parámetros por defecto?, es decir: hay lugares en los que se llama sin parámetros iniciales?

AlejoM1908 replied:
En teoría no, pero permite un poco de redundancia en el código

Commit:
Adding Responsive functionality to the views

changes:
- Adding responsive to the view

Commit:
Removing default values for access views

Changes:
- removing the default values for the access view constructor
- adding @required tag to all constructor variables
